app/views/shared/_navigation.html.erb

```diff
@@ -1,11 +1,18 @@
 <nav class="navbar navbar-expand-lg navbar-light bg-light">
-  <%= link_to("Home", home_index_path, class: "navbar-brand") %>
+  <%= render 'home/main_items'  %>
   <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
     <span class="navbar-toggler-icon"></span>
   </button>
+  &ensp;
   <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
     <div class="navbar-nav">
-        <%= link_to "Challenges", challenges_path, class: "nav-item nav-link" %>
+        <%= render 'challenges/challenges_items', class: "nav-item nav-link" %>
```

Comment:
Since the Challenges tab won't grow into something complicated, can we inline it here, rather than using the partial?